site2/docs/concepts-messaging.md

```diff
@@ -65,6 +65,7 @@ You can have different types of access modes on topics for producers.
 |---|---
 `Shared`|Multiple producers can publish on a topic. <br><br>This is the **default** setting.
 `Exclusive`|Only one producer can publish on a topic. <br><br>If there is already a producer connected, other producers trying to publish on this topic get errors immediately.<br><br>The “old” producer is evicted and a “new” producer is selected to be the next exclusive producer if the “old” producer experiences a network partition with the broker.
+`ExclusiveWithFencing`|Only one producer can publish on a topic. <br><br>If there is already a producer connected, is will be removed and invalidated immediately.
```

Comment:
```suggestion
`ExclusiveWithFencing`|Only one producer can publish on a topic. <br /><br />If there is already a producer connected, it will be removed and invalidated immediately.
```